### PR TITLE
Adding service-fabrik-postgresql-manager job for postgresql-operator

### PR DESCRIPTION
**Pull request by ankita0811**
Corresponding changes in service-fabrik-broker: https://github.com/cloudfoundry-incubator/service-fabrik-broker/pull/488

### CODE REVIEW

File: jobs/service-fabrik-postgresql-manager/spec [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+name: service-fabrik-postgresql-manager
+
+templates: 
+  bin/service-fabrik-postgresql-manager_ctl.erb: bin/service-fabrik-postgresql-manager_ctl
+  bin/job_properties.sh.erb: bin/job_properties.sh
+  config/settings.yml.erb: config/settings.yml
+  config/syslog_forwarding.conf.erb: config/syslog_forwarding.conf
+
+
```
Comment (ketakig12) [inline jobs/service-fabrik-postgresql-manager/spec]: Remove extra line.

File: jobs/service-fabrik-postgresql-manager/templates/config/settings.yml.erb [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    private_key: <%= JSON.dump(link("service-fabrik-apiserver").p('tls.apiserver.private_key')) %>
+    crds: <%= JSON.dump(link("service-fabrik-apiserver").p('crds')) %>
+      
+  ###################
+  # BACKUP SETTINGS #
+  ###################
+  backup:
+    provider: <%= JSON.dump(link("broker").p('backup.provider', nil)) %>
+
+
```
Comment (ketakig12) [inline jobs/service-fabrik-postgresql-manager/templates/config/settings.yml.erb]: Remove extra line.

File: jobs/service-fabrik-postgresql-manager/templates/bin/service-fabrik-postgresql-manager_ctl.erb [hunk 3]
```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -e # exit immediately if a simple command exits with a non-zero status
+
+source /var/vcap/packages/bosh-helpers/ctl_setup.sh 'service-fabrik-postgresql-manager'
```
Comment (subhankarc) [inline jobs/service-fabrik-postgresql-manager/templates/bin/service-fabrik-postgresql-manager_ctl.erb]: i would rather have the job name as `service-fabrik-postgresql-multitenant-operator`

We would like to change our jobs also as operators. Now that you are adding one, please make it operator.

Comment (subhankarc) [reply jobs/service-fabrik-postgresql-manager/templates/bin/service-fabrik-postgresql-manager_ctl.erb]: @jagadish-kb makes sense.

Comment (abh1kg) [reply jobs/service-fabrik-postgresql-manager/templates/bin/service-fabrik-postgresql-manager_ctl.erb]: A very naive question: why does this change need to go into the open-source repo at all? how does this help someone who wishes to deploy fabrik from the github repository?

Comment (subhankarc) [reply jobs/service-fabrik-postgresql-manager/templates/bin/service-fabrik-postgresql-manager_ctl.erb]: Considering that the agent implementation guideline SF provides, if someone adheres to that in their agent implementation, then they can use the MT operator. Right?

**Comment by ankita0811** [comment]
Closing this PR.

**Pull request closed without merging**